samples/routing/data-distribution/sample.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Data distribution
+summary: Implementing data distribution on top of NServiceBus
+component: Core
+tags:
+- Routing
+- Data distribution
+---
+
+The sample demonstrates how NServiceBus routing can be used to implement data distribution scenarios. Data distribution is about delivering a given piece of information (message) to all instances of the subscribed endpoint, as opposed to a single instance when using the Publish/Subscribe pattern built into NServiceBus. 
```

Comment:
When using the built-in Publish/Subscribe pattern in NServiceBus, all messages are delivered only to a single instance of the endpoint. In this sample every message is delivered to all endpoint instances.